hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/InflightAction.java

```diff
@@ -17,12 +17,29 @@
  */
 package org.apache.hadoop.hdds.scm.container.replication;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
This file is showing as a rename from `ReplicationActivityStatusMXBean.java → ...container/replication/InflightAction.java`, which is a bit strange. Should this InflightAction class not be a new file? Something strange has happened as you have moved things around I think.

Reply:
yea , i notice this too.  
as per our discussion , we will extract InFlightReplicaion and InFlightDeletion into a standalone class with associated test class, so i extract inflightAction a standalone class here.

i just delete ReplicationActivityStatusMXBean.java in the commit [b398eef](https://github.com/apache/ozone/pull/3352/commits/b398eef500a02867c42dbe8d613a37444137bd09)
and add InflightAction.java in another commit [f0c5034](https://github.com/apache/ozone/pull/3352/commits/f0c50340cddee2e476a6d3c918e4d86001529ff5), the options are clear in these two commits. but in the summary of Filie changed, it show `ReplicationActivityStatusMXBean.java` is moved to `InflightAction.java`,  i am not sure what happens here, do you have any idea about this?